src/clarity-angular/checkboxes/checkbox.ts

```diff
@@ -82,14 +86,17 @@ export class Checkbox implements ControlValueAccessor {
     public change = new EventEmitter<boolean>(false);
 
     public toggle() {
-        this._indeterminate = false;
```

Comment:
@youdz: I removed that line. So we are setting only the checked field from here and the checked setter takes care of the indeterminate field too through the indeterminate setter using the pattern that we discussed last week.